README.md

```diff
@@ -49,6 +50,10 @@ For a video deep dive into this repo and the samples, please see the Microsoft D
 
 [<img src="./docs/resources/deep-dive-video.jpg" width="50%">](https://youtu.be/PNykfuJ3VDs)
 
+## Docs
+
+We have published doxygen-generated documentation for all of our APIs. [Please see the link here to review how to use the APIs.](https://azure.github.io/azure-iot-middleware-freertos/)
```

Comment:
So I really don't know answer here and maybe good topic for embedded side since it has same issue, but... if i weren't familiar with doxygen I'd have no idea to find the reference material by clicking `files`.

Not sure if we should link directly to files.html.  ??

Reply:
Yea its kind of a trade off. We could link to the files but then what if they were looking for the data structures section etc. I don't mind having it this way. Luckily the top navigation bar is a relative common practice. Maybe if people have issues can revisit.